client/layout/controller.js

```diff
@@ -124,6 +127,12 @@ class Controller extends Component {
 // Update links in wp-admin menu to persist time related queries
 window.wpNavMenuUrlUpdate = function( page, query ) {
 	const search = stringifyQuery( getPersistedQuery( query ) );
+	const timeExcludedScreens = applyFilters( TIME_EXCLUDED_SCREENS_FILTER, [
+		'devdocs',
+		'stock',
+		'settings',
+		'customers',
+	] );
```

Comment:
@rrennick, you'll want to add `scheduled-actions` to this array in your plugin so that time-related queries don't get added to the url for the `Scheduled Actions` link in the sidebar. 

For example, scheduled actions sidebar link may look like this:

```
http://tangaroa.test/wp-admin/admin.php?page=wc-admin#/analytics/scheduled-actions?period=last_week&compare=previous_year
```

Reply:
Woah, @rrennick do you have a Scheduled Actions report built on this codebase?

Reply:
✨ https://github.com/Prospress/action-scheduler-admin ✨

Reply:
I also need to update it for the new page registering code.